fbchat/client.py

```diff
@@ -2400,26 +2380,29 @@ def getThreadIdAndThreadType(msg_metadata):
 
                         sticker = None
                         attachments = []
+                        deleted = False
                         if delta.get('attachments'):
                             try:
                                 for a in delta['attachments']:
                                     mercury = a['mercury']
                                     if mercury.get('blob_attachment'):
                                         image_metadata = a.get('imageMetadata', {})
                                         attach_type = mercury['blob_attachment']['__typename']
-                                        attachment = graphql_to_attachment(mercury.get('blob_attachment', {}))
+                                        attachment = graphql_to_attachment(mercury['blob_attachment'])
```

Comment:
Are you sure this change is safe? I don't remember if there was a good reason for making it like it is 😕

Reply:
Of course!
- `[]` are used one line up anyways.
- There's an `if` checking if a `'blob_attachment'` key exists, so there's no reason to 'verify' it again.
- If `.get()` would return `{}`, `graphql_to_attachment` will raise the error anyways.

Reply:
Right, I didn't even spot the `if` statement above... :grin: